olp-cpp-sdk-core/include/olp/core/client/OlpClient.h

```diff
@@ -38,50 +38,59 @@ class NetworkConfig;
 
 namespace client {
 /**
- * @brief This class is responsible for executing the REST requests.
+ * @brief Executes the REST requests.
  */
 class CORE_API OlpClient {
  public:
   OlpClient();
   virtual ~OlpClient() = default;
 
   /**
-   * @brief Set the base url used for all requests.
-   * @param baseUrl Base url.
+   * @brief Sets the base URL used for all requests.
+
+   * @param baseUrl The base URL.
    */
   void SetBaseUrl(const std::string& baseUrl);
 
   /**
-   * @brief Get the base url.
+   * @brief Gets the base URL.
+   *
+   * @return The base URL.
    */
   const std::string& GetBaseUrl() const;
 
   /**
-   * @brief Get the default headers that is added for each request.
+   * @brief Gets the default headers that are added to each request.
+   *
+   * @return The default headers.
    */
   std::multimap<std::string, std::string>& GetMutableDefaultHeaders();
 
   /**
-   * @brief Set the client settings.
-   * @param settings Client settings.
+   * @brief Sets the client settings.
+   *
+   * @param settings The client settings.
    */
   void SetSettings(const OlpClientSettings& settings);
 
   /**
-   * @brief Execute the REST request through the network stack
-   * @param path Path that is appended to the base url.
-   * @param method Choice of GET, POST, DELETE, PUT.
-   * @param query_params Params that is appended to the path url.
-   * @param header_params Headers to customize request.
-   * @param form_params For a POST request, form_params or post_body should be
-   * populated, but not both.
-   * @param post_body For a POST request, form_params or post_body should be
-   * populated, but not both.
-   * This data must not be modified until the request is completed.
-   * @param content_type Content type for the post_body or form_params.
-   * @param callback a function callback to receive the HttpResponse.
+   * @brief Executes the REST request through the network stack.
```

Comment:
```suggestion
   * @brief Executes the HTTP request through the network stack.
```
Not related to the change, as the previous version of the docs was not entirely correct.

Reply:
Changed